### PR TITLE
Add mime type to workflow input files

### PR DESCRIPTION
**Pull request by berggren**
This PR adds the attribute magic_mime so workers can get the information.

### CODE REVIEW

File: src/api/v1/workflows.py [hunk 1]
```diff
@@ -239,6 +239,7 @@ async def run_workflow(
             "display_name": file.display_name,
             "extension": file.extension,
             "data_type": file.data_type,
+            "mime_mime": file.magic_mime,
```
Comment (hacktobeer) [inline src/api/v1/workflows.py:242]: mime_type?

**Review by hacktobeer — CHANGES_REQUESTED** [review]
typo?

File: src/api/v1/workflows.py (hunk 1 above)
Comment (berggren) [reply src/api/v1/workflows.py]: fixed

**Review by jleaniz — APPROVED** [review]
LGTM

**Pull request merged**